django/contrib/admin/static/admin/css/base.css

```diff
@@ -515,7 +516,7 @@ select[multiple] {
     padding: 10px 15px;
     border: none;
     border-radius: 4px;
-    color: var(--button-fg);
+    color: var(--body-fg);
```

Comment:
Check out [comment](https://github.com/django/django/pull/16268/files#discussion_r1217213838).

Reply:
Shouldn't we instead change a `button-bg`? This will not fix contrast issues in the other places where the `button-bg` is used, e.g. on pagination, and it's not enough for hover:
![image](https://github.com/django/django/assets/2865885/6b99c7ef-6ff4-4fea-833d-9ad8cc807f5b)